packages/app/src/components/editor/ArticleWizard.tsx

```
@@ -0,0 +1,404 @@
+            </div>
+
+            <textarea id="article-content-textarea" value={content} onChange={(e) => setContent(e.target.value)} placeholder="Write your article in markdown..." className="w-full h-80 p-2 text-sm border border-neutral-200 rounded bg-transparent" />
+          </div>
+        )}
+
+        {step === 2 && (
+          <div>
+            <h3 className="text-lg font-medium">Preview</h3>
+            {coverImage && <img src={coverImage} alt="cover" className="w-full h-40 object-cover rounded mt-2" />}
+            <h2 className="mt-2 text-2xl font-heading">{title || "Untitled"}</h2>
+            <div className="prose max-w-none whitespace-pre-wrap mt-2" dangerouslySetInnerHTML={{ __html: renderMarkdownToHtml(content) }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)